jekyll/_cci2/pipeline-variables.adoc

```diff
@@ -27,6 +27,8 @@ Pipeline values and parameters can be used to create reusable pipeline configura
 
 Pipeline values are available to all pipeline configurations and can be used without previous declaration.
 
+Please note: if using setup workflows, all pipeline parameters must be present in the subsequent configuration, even if calling with the API 
```

Comment:
I think I'd actually go for moving this note in a section further down the page - I'll add it as a suggestion :)